plugins/pulp_docker/plugins/migrations/0004_tag_schema_change.py

```diff
@@ -0,0 +1,20 @@
+from pulp.server.db.connection import get_collection
+
+
+def migrate(*args, **kwargs):
+    """
+    Add manifest_type to the tag collection.
+    """
+
+    manifest_type_key = 'manifest_type'
+    collection = get_collection('units_docker_tag')
+    # drop old index due to unit_keys fields change
+    index_info = collection.index_information()
+    old_index = 'name_1_repo_id_1_schema_version_1'
+    if old_index in index_info:
+        collection.drop_index(old_index)
+    # update collection with new field
+    for tag in collection.find():
```

Comment:
I think you could do this whole thing on the DB side with one update call, rather than iterate through the items in python.